UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -416,6 +426,63 @@ public virtual async Task<Material> LoadMaterialAsync(int materialIndex)
 			return _assetCache.MaterialCache[materialIndex].UnityMaterialWithVertexColor;
 		}
 
+        /// <summary>
+        /// Load a Mesh from the glTF by index
+        /// </summary>
+        /// <param name="meshIndex"></param>
+        /// <returns></returns>
+        public virtual async Task<Mesh> LoadMeshAsync(int meshIndex, CancellationToken cancellationToken)
+        {
+            try
+            {
+                lock (this)
+                {
+                    if (_isRunning)
+                    {
+                        throw new GLTFLoadException("Cannot CreateMesh while GLTFSceneImporter is already running");
+                    }
+
+                    _isRunning = true;
+                }
+
+				if (_options.ThrowOnLowMemory)
```

Comment:
A lot of this boilerplate is shared in common with LoadSceneAsync, LoadMaterialAsync, and LoadNodeAsync.  Consider wrapping it in a Setup() function that lives next to Cleanup().

Specifically, constructing _memoryChecker and _assetCache, and calling LoadJson().

Reply:
Good call! Done.